.rubocop.yml

```diff
@@ -43,6 +43,7 @@ Naming/FileName:
 Naming/AccessorMethodName:
   Exclude:
     - "lib/nylas/scheduler.rb"
+    - "lib/nylas/scheduler_collection.rb"
```

Comment:
Just checking: is this a rule we want to keep enforcing, or just make an exception in this case?

Reply:
I'm indifferent about this rule honestly, I'll just get rid of it